.github/workflows/test.yml

```diff
@@ -35,11 +35,11 @@ jobs:
           toolchain: stable
           override: true
 
-      - uses: Swatinem/rust-cache@v2
-        with:
-          workspaces: |
-            .
-            bindings_ffi
+      # - uses: Swatinem/rust-cache@v2
```

Comment:
The CI is somehow running out of disk space. I'm working through more performant solutions, but for now this has made the GH checks pass.